pkg/operator/ceph/config/defaults.go

```diff
@@ -93,8 +93,15 @@ func DefaultCentralizedConfigs(cephVersion version.CephVersion) []Option {
 // made to override these options for the Ceph clusters it creates.
 func DefaultLegacyConfigs() []Option {
 	overrides := []Option{
-		// TODO: drop this when FlexVolume is no longer supported
+		// TODO: move this under LegacyConfigs() when FlexVolume is no longer supported
 		configOverride("global", "rbd_default_features", "3"),
 	}
 	return overrides
 }
+
+// LegacyConfigs represents old configuration that were applied to a cluster and not needed anymore
+func LegacyConfigs() []Option {
+	return []Option{
+		{Who: "global", Option: "log file"},
```

Comment:
How about a comment that indicates in which release we can remove this legacy setting removal? We should only need to keep it for a single minor release. For example, since it's going in v1.6.x, we could remove this in v1.7, right?

Reply:
or after this one is merged/backported, we could even immediately open the master PR to remove it

Reply:
I'll just add it to https://github.com/rook/rook/issues/7183 to do it in one PR.